src/whylogs/core/metrics/nlp_metrics.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Union
+
+import jiwer
```

Comment:
We decided to try / import for this in our meeting.

Reply:
it was done